lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -201,6 +201,12 @@ class StandardAccountTest {
         assertNotNull(feeEstimate)
     }
 
+    @Test
+    fun `estimate message fee`() {
+        // TODO: test provider.getEstimateMessageFee
+        // BLOCKED: pending devnet update
+    }
+
```

Comment:
any idea when devnet will be updated?
how were your changes tested?

Reply:
Unfortunately, no idea. They weren't tested so probably should be blocked until the testing is possible.

Reply:
I see. We could wait, but I can't even see a PR for this in devnet and it would be a shame to hold on for who knows how long.

My suggestion is to mock this test case for now. If you want to, you can even run this against testnet to see what is an example response. For some example on ocked tests, you can check ProviderTests. To get rid of this mocks, let's create a new issue and deal with it when devnet support is in place.

What do you think about this?